### PR TITLE
Add terra.json files to each workflow.

### PR DESCRIPTION
**Pull request by wnojopra**
A terra.inputs.json and a terra.outputs.json file that you can directly upload to Terra is now included for each workflow.

### CODE REVIEW

File: rna/update-samples-entity/update-samples-entity-from-samples-list.terra.inputs.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "SampleToPairedFastq.mapping_file": "${}",
```
Comment (wnojopra) [inline rna/update-samples-entity/update-samples-entity-from-samples-list.terra.inputs.json:6]: Matt, I left this purposefully blank. What do you think?

**Review by mbookman — CHANGES_REQUESTED** [review]
Can we structure these Terra inputs.json files to be similar to the non-Terra inputs.json, with grouping and comments?

File: rna/rna-alignment/rna-alignment.terra.inputs.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "RNAAlignment.star_index": "gs://amp-pd-transcriptomics/inputs/reference/star/STAR_genome_GencodeV29_oh125.tar.gz",
+
+  "##_COMMENT3": "Path to a Docker image containing star",
+  "RNAAlignment.star_docker": "alexdobin/star:2.6.1d",
+
+  "RNAAlignment.runtime_zones": "us-central1-a us-central1-b",
+  "RNAAlignment.star_vm_memory": "80.0GB",
+  "RNAAlignment.star_vm_disk_size_gb": "${200}",
+
+  "##_COMMENT4": "STAR does not *typically* take more than 24 hours, so use preemptible",
```
Comment (mbookman) [inline rna/rna-alignment/rna-alignment.terra.inputs.json:17]: This comment doesn't match the preemptible_tries value.

File: rna/rna-quantification/rna-quantification.terra.outputs.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "RNAQuantification.logs": "${this.salmon_logs}",
```
Comment (mbookman) [inline rna/rna-quantification/rna-quantification.terra.outputs.json]: There's no obvious structure here, so could you sort these based on the name of the variable?

File: rna/update-samples-entity/update-samples-entity-from-samples-list.terra.inputs.json (hunk 1 above)
Comment (mbookman) [reply rna/update-samples-entity/update-samples-entity-from-samples-list.terra.inputs.json:6]: That seems right to me. Can you have the comment say "GCS Path to the mapping tsv file"?

**Review by mbookman — CHANGES_REQUESTED** [review]

**Review by mbookman — APPROVED** [review]
LGTM

**Review by ha-duong-technome — APPROVED** [review]

**Pull request merged**